### ISSUE TITLE
DOI in Hydranorth

### ISSUE THREAD

**Issue by weiweishi** [ualbertalib/HydraNorth#448]
Following the initial discussion here:https://groups.google.com/forum/#!topic/hydra-tech/oBnDYP4hheE, 
ideas have been bounced around for the use of DOI in Hydranorth:
More discussions should happen before the production launch regarding our strategy. 
1. should we mint DOIs for objects deposited into Hydranorth
2. should we present user the option to enter DOI for existing/new deposited items if they have a DOI for the object?
3. We need to look into what's needed to maintain DOIs

We want to avoid getting bogged down in the complexity of bibliographic relationships. Users want DOIs and we need to start giving them DOIs soon.

**Comment by johnhuck** [comment]
Since all resources in Dataverse have DOIs assigned as part of normal deposit actions, we will need to ensure there is a way to avoid assigning double DOIs when Dataverse metadata is ingested into HydraNorth.

**Comment by sfarnel** [comment]
It will be good to sort through the question of double DOIs as this will be
very relevant for other items that may be deposited, in particular journal
articles.

On Mon, Jul 6, 2015 at 1:32 PM, johnhuck <notifications@github.com> wrote:

> Since all resources in Dataverse have DOIs assigned as part of normal
> deposit actions, we will need to ensure there is a way to avoid assigning
> double DOIs when Dataverse metadata is ingested into HydraNorth.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/448#issuecomment-118971319>
> .
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by pbinkley** [comment]
I'm doubtful that we will get to DOI-minting in HydraNorth until after the
launch, so this isn't likely to be a problem during migration. It shouldn't
be a problem though to have the DOI-minting process check whether an item
already has a DOI and avoid adding a second one. (Of course, it will depend
on the original DOI being present in the imported metadata.)

​Peter​

**Comment by johnhuck** [comment]
Yes, the DOI is present in the metadata and it is the link that takes users back to Dataverse to access the resource. It seems like this issue is one that can be parked until we are closer to minting DOIs in HydraNorth.

**Comment by wyman10** [comment]
There is also the issue of a DOI being generated before a study is released. So any checking would have to occur on DOI's for these studies in DV because the metadata will not be would in HydraNorth.

**Comment by pbinkley** [comment]
Hmm, interesting. Pardon my ignorance, but is there anything built into
Dataverse's workflows to prevent the issuing of a new local DOI for items
that already have an externally generated one?

Peter

Peter Binkley, Ph.D., MLIS
Digital Initiatives Technology Librarian
Information Technology Services
peter.binkley@ualberta.ca

2-10K Cameron Library
University of Alberta
Edmonton, Alberta
Canada T6G 2J8

phone 780-492-3743
fax 780-492-9243

**Comment by wyman10** [comment]
The DOIs in Dataverse are resolved via the EZID service - so when a study is created a DOI is generated automatically and stays with that study even it is never released. I can not recall how many DOIs UAL purchased? But there is a pool of them that we are drawing from via EZID.

**Comment by pbinkley** [comment]
​Ah, I misunderstood - it's not a case of data coming to Dataverse with
pre-existing DOIs from some other source. Does that ever happen?

Peter​

Peter Binkley, Ph.D., MLIS
Digital Initiatives Technology Librarian
Information Technology Services
peter.binkley@ualberta.ca

2-10K Cameron Library
University of Alberta
Edmonton, Alberta
Canada T6G 2J8

phone 780-492-3743
fax 780-492-9243

**Comment by wyman10** [comment]
Not currently. At the moment everything in Dataverse is created in dataverse (1st rule of Dataverse). Down the line we may harvest metadata for other studies or have a power user who wants to move a bunch of data files that already have metadata, and an associated DOI - at that point we would have to ensure a workflow that would not "stamp" or "replace" the DOI associated with a study's metadata that is being imported into DV. 

There has been some talk on CANLIB-DATA relating to Stats Can files that are being ingested into multiple dataverses and as such getting a unique handle at UBC and another at SP. The consensus seems to be that is all right because it is pointing to a digital object residing in a particular repository.

**Comment by weiweishi** [comment]
DOI is now in HN! Close the ticket with :beer:
